xwalk_android.gypi

```diff
@@ -213,6 +213,59 @@
       'includes': ['../build/jni_generator.gypi'],
     },
     {
+      'target_name': 'xwalk_runtime_lib_lzma_compress',
+      'type': 'none',
+      'dependencies': [
+        'xwalk_runtime_lib_apk',
+      ],
+      'variables': {
+        'files_to_compress': [
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/stripped_libraries/libxwalkcore.so',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib_apk/classes.dex',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/icudtl.dat',
+          '<(PRODUCT_DIR)/xwalk_runtime_lib/assets/xwalk.pak',
+        ],
+        'assets_dir': '<(PRODUCT_DIR)/xwalk_runtime_lib_lzma/assets/',
+      },
+      'actions': [
+        {
+          'action_name': 'lzma_compress',
+          'message': 'compress ingredients for runtime APK',
+          'inputs': [
+            'build/android/lzma_compress.py',
+          ],
+          'outputs': [
```

Comment:
I might be wrong, but if you do this you risk making `ninja -t clean` fail and if any other target depends on the files generated here they might not work since those files won't be known to gyp. In general, it is better to register all files that your action ends up creating.